website/docs/commands/validate.html.markdown

```diff
@@ -24,10 +24,31 @@ The following can be reported:
  * invalid `module` name
  * interpolation used in places where it's unsupported
  	(e.g. `variable`, `depends_on`, `module.source`, `provider`)
+ * missing value for a variable (none of `-var foo=...` flag,
+   `-var-file=foo.vars` flag, `TF_VAR_foo` environment variable,
+   `terraform.tfvars`, or default value in the configuration)
 
 ## Usage
 
-Usage: `terraform validate [dir]`
+Usage: `terraform validate [options] [dir]`
 
 By default, `validate` requires no flags and looks in the current directory
-for the configurations.
\ No newline at end of file
+for the configurations.
+
+The command-line flags are all optional. The available flags are:
+
+* `-config-only` - If specified, the command will check basic syntax of the
```

Comment:
Doh, I did `grep` through the code, but only in `command/*`, will fix that in a minute.